src/App.css.orig

```diff
@@ -0,0 +1,11 @@
+.background {
+
+    height: 100vh;
+    width: 100%;
+    text-align: center;
+    background-image: url(./assets/kv_bgimg.jpg);
+    background-size: cover;
```

Comment:
the image should not get squished when resizing the width of the browser.
it might be the `background-size` property 🤔

Reply:
not sure what you mean

Reply:
the coconut image doesn't resize to scale. it gets squished.

![image](https://user-images.githubusercontent.com/46483718/134106769-7aa49db2-522d-4528-9572-98b44089044e.png)

Reply:
Changed it to resize to scale